docs/site/Authentication-component-decorator.md

```diff
@@ -18,15 +18,25 @@ functions with the
 
 The decorator's syntax is:
 
-```ts
-@authenticate(strategyName: string, options?: object)
-```
-
-or
-
-```ts
-@authenticate(metadata: AuthenticationMetadata)
-```
+- single strategy: `@authenticate(strategyName)`
```

Comment:
Maybe we can group the single strategy with and without options together? likewise for the multiple strategies?

Reply:
thanks @dhmlau changed.